configuration/steps/commands/infra.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, container_name: str, runtime_tag: str, step_name: str):
     def as_cmd_arg(self) -> list[str]:
         return [
             "bash",
-            "-c",
+            "-exc",
             (
                 "docker container commit "
                 f"""--message "{self.step_name}" {self.container_name} """
```

Comment:
I hadn't notice this before, so a small suggestion:
You know you can do:
`f'--message "{self.step_name}" {self.container_name} '`
instead of using the `"""` triplets?